book/getting-started.rst

```diff
@@ -0,0 +1,140 @@
+Getting Started
+===============
+
+Are you ready for a quick start with Sulu? Sit back, fasten your seat belts
+and... go!
+
+Bootstrap a Project
+-------------------
+
+We'll bootstrap a new project based on the `Sulu Minimal Edition`_ with
+Composer_:
+
+.. code-block:: bash
+
+    composer create-project sulu/sulu-minimal my-project -n
+
+This command will bootstrap a new project in the directory ``my-project``.
+
+.. tip::
+
+    Now is a good time to start versioning your project. If you use Git,
+    initialize a new Git repository and submit your first commit:
+
+    .. code-block:: bash
+
+        cd my-project
+        git init
+        git add .
+        git commit -m "Initial commit"
+
+Webspaces
+---------
+
+The content management part of Sulu is built upon *webspaces*. Each of these
+webspaces configure a content tree. Each content tree may contain translations
+for different locales.
+
+The default webspace configuration is located in
+``app/Resources/webspaces/example.com.xml``. Rename this file so that it matches
+the name of your project.
+
+To get started, you only need to modify the ``<key>`` in that file. Change it
```

Comment:
Maybe we should also mention `<name>` here, and explain that it is the text being displayed in the UI?